src/services/UserService.js

```diff
@@ -22,4 +23,24 @@ export default class UserService {
     userCredentials.password = await Helper.hashPassword(password);
     return User.create(userCredentials);
   }
+
+  /**
+   * @method signin
+   * @description Signs in user with valid credentials
+   * @static
+   * @param {object} loginCredentials - data object
+   * @returns {object} JSON response
+   * @memberof UserService
+   */
+  static async signin(loginCredentials) {
+    const { email } = loginCredentials;
```

Comment:
You can trim the `email` and convert to lowercase
```suggestion
    let { email } = loginCredentials;
    email = email.trim().toLowerCase();
```

Reply:
Correction received, thanks